assets/questions/findChar.js

```diff
@@ -0,0 +1,114 @@
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Question data for Reverse Words.
+ */
+
+globalData.questions['findChar'] = { // eslint-disable-line dot-notation
+  title: 'Find the first character in a string that doesn\'t repeat',
+  starterCode: {
+    python: `def find1stNonRepeatedChar(word):
+    return ""
+`
+  },
+  auxiliaryCode: {
+    python: `class AuxiliaryCode(object):
+    @classmethod
+    def sortChar(cls, word):
+        sortedStr = ''.join(sorted(word))        
+        c = [a for a in sortedStr if sortedStr.count(a) == 1]
+        if not c:
+            return None
+        else:
+            return c[0]
+`
+  },
+  tasks: [
+    {
+      instructions: [
+        [
+          'For this question, you\'ll implement the find1stNonRepeatedChar function. ',
+          'Write a function to find the first character in a String that doesn\'t repeat.'
+        ].join(''),
+        'For instance, \'mom\' would output \'o\'.'
+      ],
+      prerequisiteSkills: ['Strings', 'String Manipulation'],
+      acquiredSkills: ['String Manipulation'],
+      inputFunctionName: null,
+      outputFunctionName: null,
+      mainFunctionName: 'find1stNonRepeatedChar',
+      correctnessTests: [
+        {input: 'mom', 
+          allowedOutputs: ['o']},
+        {input: 'apple', 
+          allowedOutputs: ['a']},
+        {input: 'wefffa', 
+          allowedOutputs: ['w']},
+        {input: 'weffwa', 
+          allowedOutputs: ['e']},
+        {input: '', 
+          allowedOutputs: [null]},
+        {input: 'aaaeee', 
+          allowedOutputs: [null]}
+      ],
+      buggyOutputTests: [{
+        buggyFunctionName: 'AuxiliaryCode.sortChar',
+        messages: [[
+          'Try running your code on \'wefffa\' on paper. What\'s the result?   ',
+          'Are you sure that you\'re returning the first non-repeating character in the word?   ',
+          'It looks like you\'re returning the character that is unique but not the first one in the given string.'
+        ].join('')]
+      }],
+      performanceTests: []
+    },
+    {
+      instructions: [[
+        'Next, double-check your code to make sure that it handles ',
+        'more than just letters.'
+      ].join('')],
+      prerequisiteSkills: ['Arrays', 'Strings', 'String Manipulation'],
+      acquiredSkills: ['String Manipulation', 'Sets', 'Arrays', 'Maps'],
+      inputFunctionName: null,
+      outputFunctionName: null,
+      mainFunctionName: 'find1stNonRepeatedChar',
+      correctnessTests: [
+        {input: 'TT AsAs', 
+          allowedOutputs: [' ']},
+        {input: 'AbCABcBb', 
+          allowedOutputs: ['C']},
+        {input: 'BB337 7122 ', 
+          allowedOutputs: ['1']},
+        {input: 'test11@test', 
+          allowedOutputs: ['@']}
+      ],
+      buggyOutputTests: [],
+      performanceTests: [{
```

Comment:
This doesn't seem like a good inputDataAtom for a performance test. You may need to write your own transformation function, because otherwise it'll just duplicate the string a lot and every character will end up repeated. Also, I'd imagine there shouldn't be a space.